### PR TITLE
v1.30 release

### PR DESCRIPTION
**Pull request by plondon**

### CODE REVIEW

**Comment by rogerwhitt** [comment]
what the hell is all this

**Comment by plondon** [comment]
Hi @rogerwhitt the majority of this code introduces our SFOX partner integration for trading btc.

**Comment by rogerwhitt** [comment]
i am 70 years old and demented all i want is my 700.00 that was taken from my bovada poker accoun 

    On Wednesday, January 10, 2018, 11:47:10 AM EST, Philip London <notifications@github.com> wrote:  
 
 
Hi @rogerwhitt the majority of this code introduces our SFOX partner integration for trading btc.

File: app/partials/sfox/link.pug [hunk 1]
```diff
@@ -28,7 +28,7 @@
                 span.f-14.capitalize {{ bankAccount.institution_type }} Bank
         .group.full.mb-10
           .item
-            label Name on Account
+            label Account Holder's Name
```
Comment (Bia296) [inline app/partials/sfox/link.pug]: Beatriz de Macedo

File: app/partials/sfox/link.pug [hunk 2]
```diff
@@ -28,7 +28,7 @@
                 span.f-14.capitalize {{ bankAccount.institution_type }} Bank
```
Comment (Bia296) [inline app/partials/sfox/link.pug]: Banco Bradesco

**Comment by rogerwhitt** [comment]
gezz where is my money

    On Wednesday, January 10, 2018, 12:20:33 PM EST, Bia296 <notifications@github.com> wrote:  
 
 
@Bia296 commented on this pull request.

In app/partials/sfox/link.pug:
> @@ -28,7 +28,7 @@
                 span.f-14.capitalize {{ bankAccount.institution_type }} Bank

Banco Bradesco

**Pull request merged**